pkg/providers/aws/cluster_network_provider_test.go

```diff
@@ -1835,7 +1834,7 @@ func TestNetworkProvider_GetClusterNetworkPeering(t *testing.T) {
 			args: args{
 				ctx: context.TODO(),
 			},
-			wantErr: "failed to get standalone vpc: error getting vpcs: ec2 get vpcs error",
+			wantErr: true,
```

Comment:
Quick question regarding these changes, is this just for consistency with other tests? even though it's a big strange comparing the error string I guess these ensure that we get the desired error rather than any error

Reply:
Yes I made that change for consistency and because comparing strings seemed wrong. But I see what you mean, comparing the error strings gives us a more specific test. I'll revert those changes.